src/dune/args.py

```diff
@@ -84,6 +84,12 @@ def __init__(self):
                                            "PRIV_KEY (Optional)"],
                                   help='create an EOSIO account and an optional creator (the '
                                        'default is eosio)')
+        self._parser.add_argument('--system-newaccount', nargs='+',
```

Comment:
I think `nargs='+'` is inappropriate here since haven't done anything to parse the metavars multiple times. Probably should be `nargs='*'`, but I'm not 100% sure.

Reply:
Here's the problem with `'*'`:
```
  --system-newaccount
   ['NAME', 'CREATOR (Optional)', 'PUB_KEY (Optional)', 'PRIV_KEY (Optional)', '-- FLAGS (Optional)']
   [['NAME', 'CREATOR (Optional)', 'PUB_KEY (Optional)', 'PRIV_KEY (Optional)', '-- FLAGS (Optional)']
      ...]
```

The help doesn't match the behavior. For example, this doesn't seem to work:
```
dune --system-newaccount mytok sb <pubkey> <priv key> mytok2 sb <pubkey> <privkey>
```

I think we'd be better off having `--help` provide correct instruction and then just checking the arg count in `__main__.py`. We don't want to lie to our users, right?

Reply:
Ah, nice catch.
In terms of `--help` there is only a slight difference between `'+'` and `'*'`.
It seems we were using nargs incorrectly in all places, so I have created an issue for it #123 
I don't see an easy solution for it.
It seems the proper way would be to have separate options for each argument so: `--system-newaccount-name --system-newaccount-creator --system-newaccount-pub-key --system-newaccount-priv-key`

Reply:
Issue #123 was written to address this.